app/views/application/_forfeit_game.html.erb

```diff
@@ -0,0 +1,22 @@
+<!-- Login Modal -->
+<!-- <div id="forfeitModal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title">End Game</h4>
+        <button type="button" class="close-icon modal-close" data-modal-to-close="#forfeitModal"></button>
+      </div>
+      <div class="modal-body">
+      <p>Are you sure you want to forfeit the game?</p>   
+      </div>
+      <div class="modal-footer">
+          <ul class="form-footer-links-and-statuses">
+            <div class="form-footer-status">
+            </div>
+          </ul>
+          <button id="endGame" class="custom-modal-action btn btn-default" data-ajax-method="PATCH" data-ajax-data="" data-ajax-url="/games/2/forfeit">End Game</button>
```

Comment:
Ah woops I think Harsheek and I forgot to make it not hardcoded after getting the functionality working. Should be something like `data-ajax-url="/games/<%= @game.id %>/forfeit"`. Or I guess a ruby content tag for the whole button would work as well, might be cleaner. In games/show.html, the `@game` instance variable will have to be passed where there's `<%= render partial: 'forfeit_game' %>`. I think it's something like `<%= render partial: 'forfeit_game', locals: @game %>`.